src/App.tsx

```diff
@@ -16,46 +16,120 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [isReversed, setIsReversed] = useState(false);
+  const [sortedBy, setSortedBy] = useState('none');
+  const [isVisible, setIsVisible] = useState(false);
+
+  const changeArray = (array: string[]) => {
+    const visibleGoods = [...array];
+
+    switch (sortedBy) {
+      case 'none':
+        break;
+
+      case 'alphabet':
```

Comment:
Don't hardcode the cases, create Enum for them. Also, you can combine none and default cases

Reply:
done